runtime/compiler/control/CompilationThread.cpp

```diff
@@ -8471,8 +8471,9 @@ TR::CompilationInfoPerThreadBase::wrappedCompile(J9PortLibrary *portLib, void *
                   }
 
                // See if we need to insert GCR trees
-               if (!details.supportsInvalidation() ||
-                   options->getOptLevel() >= hot) // Workaround for a bug with GCR inserted in hot bodies. See #4549 for details.
+               //if (!details.supportsInvalidation() ||
```

Comment:
This surprises me - did you intend to include this change with everything else it doesn't necessarily seem related??

Reply:
My bad, I accidentally committed it to this PR. I used this change to verify the fix in the reported issue. Will remove it.